snippets/kubernetes.json

```diff
@@ -460,5 +460,834 @@
+            "      - name: ${4:name}",
+            "        image: ${5:nginx}",
+            "        resources:",
+            "          requests:",
+            "            cpu: 100m",
+            "            memory: 200Mi",
+            "        ports:",
+            "        - containerPort: ${6:8080}",
+            "---",
+            "apiVersion: v1",
+            "kind: Service",
+            "metadata:",
+            "  name: $1",
+            "  namespace: $2",
+            "spec:",
+            "  ports:",
+            "  - port: ${7:80}",
+            "    targetPort: $6",
+            "  selector:",
+            "    $3",
+            "---",
+            "apiVersion: networking.k8s.io/v1",
+            "kind: Ingress",
+            "metadata:",
+            "  annotations:",
+            "    nginx.ingress.kubernetes.io/rewrite-target: /",
+            "    kubernetes.io/ingress.class: nginx",
+            "    cert-manager.io/cluster-issuer: \"letsencrypt-prod\"",
+            "  name: $1",
+            "  namespace: $2",
+            "spec:",
+            "  ${10:tls:",
+            "  - secretName: ${9:$8.tls}",
+            "    hosts:",
+            "    - $8",
+            "  }rules:",
+            "  - host: ${8:host}",
+            "    http:",
+            "      paths:",
+            "      - path: ${11:/}",
+            "        pathType: Prefix",
+            "        backend:",
+            "          service:",
+            "            name: $1",
+            "            port:",
+            "              number: $7"
+        ],
+        "prefix": "depsvcing"
+    },
+    "SecretKeyRef": {
+        "body": [
+            "valueFrom:",
+            "  secretKeyRef:",
+            "    name: ${1:secret}",
+            "    key: ${2:key}"
+        ],
+        "prefix": "skr"
+    },
+    "ServiceAccount": {
+        "body": [
+            "apiVersion: v1",
+            "kind: ServiceAccount",
+            "metadata:",
+            "  name: ${1:name}"
+        ],
+        "prefix": "sa"
+    },
+    "DaemonSet": {
+        "body": [
+            "apiVersion: apps/v1",
+            "kind: DaemonSet",
+            "metadata:",
+            "\tname: ${1:name}",
+            "\tnamespace: ${2:default}",
+            "\tlabels:",
+            "\t\tapp: $1",
+            "spec:",
+            "\tselector:",
+            "\t\tmatchLabels:",
+            "\t\t\tapp: $1",
+            "\ttemplate:",
+            "\t\tmetadata:",
+            "\t\t\tlabels:",
+            "\t\t\t\tapp: $1",
+            "\t\tspec:",
+            "\t\t\tcontainers:",
+            "\t\t\t- name: $1",
+            "\t\t\t\timage: ${3:image/name}",
+            "\t\t\t\timagePullPolicy: IfNotPresent",
+            "\t\t\t\tresources:",
+            "\t\t\t\t\tlimits:",
+            "\t\t\t\t\t\tcpu: 100m",
+            "\t\t\t\t\t\tmemory: 100Mi",
+            "$0"
+        ],
+        "prefix": "ds"
+    },
+    "Ingress TLS section": {
```

Comment:
![image](https://user-images.githubusercontent.com/56930510/176342824-f102de62-c774-4ea8-b104-131861776064.png)

What's the use of your snippet (`ingtls`) versus just using this `k-ingress-tls` ? Just making sure we aren't repeating ourselves is all :smile: